cmd/guardian/main.go

```diff
@@ -60,6 +61,9 @@ var rootCmd = func() cli.Command {
 			"detect-iam-drift": func() cli.Command {
 				return &drift.DetectIamDriftCommand{}
 			},
+			"iam-cleanup": func() cli.Command {
```

Comment:
as a follow up, it would be nice to co locate these as subcommands to the `drift` command

e.g. `guardian drift iam-cleanup` and `guardian drift iam-detect`

Reply:
But this tool isn't explicitly about drift. It can be used to delete any IAM.

Reply:
Why would someone run IAM cleanup without having run drift detection?

Reply:
Because they are using AOD and old AOD requests don't necessarily get cleaned up. Or maybe because they would like to remove access from an org for a particular User/Group/ServiceAccount for all org/folders/projects.

Reply:
Or could be `cli iam detect-drift` and `cli iam cleanup`

Reply:
I like Chen's proposal, as these are specific to IAM

Reply:
Will do it in a follow up